.github/workflows/inductor_xpu_e2e_nightly.yml

```diff
@@ -41,6 +41,16 @@ on:
         type: string
         default: ''
         description: If set, will only launch this one
+      torch_xpu_ops_update:
+        required: false
+        type: string
+        default: 'true'
+        description: True means update xpu_ops when building pytorch, otherwise means not
+      test_ut:
+        required: true
+        type: string
+        default: 'true'
+        description: True will running UT test for Ondemand test, false will not.
 
```

Comment:
Can we also add pytorch branch in here also?

Reply:
Done